apps/conduit-e2e/src/integration/new-article/new-article.ts

```diff
@@ -0,0 +1,57 @@
+import { Given, When, And, Then } from 'cypress-cucumber-preprocessor/steps';
+import { generateRandomString } from '../../support/generate-random-string';
+
+let userId = '';
+let articleTitle = '';
+
+beforeEach(() => {
+  userId = generateRandomString();
+  articleTitle = generateRandomString();
+});
+
+Given('I am logged in to the system', () => {
+  cy.loginApi(userId);
+});
+
+And('I am on New Post page', () => {
+  cy.visit('#/editor');
+});
+
+When('I input the title of the article', () => {
+  cy.get("[placeholder='Article Title']").clear().type(articleTitle);
+});
+
+And('I input the summary of the article', () => {
+  cy.get('[placeholder="What\'s this article about?"]').clear().type('How to do automation testing');
+});
+
+And('I input the body of the article', () => {
+  cy.get("[placeholder='Write your article (in markdown)']").clear().type('Automation testing description');
+});
+
+And('I input tags of the article', () => {
+  cy.get("[placeholder='Enter Tags']").clear().type('testing');
```

Comment:
I understand why you had to use "[placeholder=..." for all these fields. I'll change the form components in order to be able to locate them with the getByE2eId.